Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM fuzzers/afl:2.52
```

Comment:
A few things: why are we cloning `oiio` instead of copying it in? The checkout phase already happens in the action. The two other clone commands look strange as well, is there no way to install these dependencies from a package manager? Also, why are we linking these header files? Finally, it's best practice to include a package stage at the end (especially since it seems you have installed a lot of dependencies).

Reply:
Refactored.
Btw, my fork was not sync and was old. It should be updated and ready to pull now.